mRides-app/MainActivity.cs

```diff
@@ -142,13 +148,24 @@ protected override void OnResume()
         protected override void OnCreate(Bundle bundle)
         {
             base.OnCreate(bundle);
-            SetContentView(Resource.Layout.Main);
+            var account = AccountStore.Create(this).FindAccountsForService("Facebook").FirstOrDefault();
+            //IEnumerable<Account> accounts = AccountStore.Create(this).FindAccountsForService("Facebook");
```

Comment:
Is this commented line of code necessary since we have the one above? If not, it is better to remove it in case you will forget.